### ISSUE TITLE
Reddening correction and density of stars in the Milky Way band

### ISSUE THREAD

**Issue by chengsoonong** [chengsoonong/mclass-sky#27]
Correct for the different distortions caused by dust in different parts of the sky (dec, ra). Dust causes reddening, and we can download the correction vector (with a transformation) from SDSS.

**Comment by chengsoonong** [comment]
The class prior probabilities also change with location in the sky. Stars are more common in a certain band (Milky way).

**Comment by alasdairtran** [comment]
With the correction for dust extinction, there are three sets of correction. Apply each of them (subtract them from the magnitudes) and plot the heat map.

Here's a prediction. Before the correction, the classes diffuse in the feature space. They are smoothed out, with a lot of overlap. Now the best correction should unstretch and move the classes back in the right direction, giving us a more compact set of classes. That is, we should get the least amount of class ambiguity afterwards.

But how do we measure class ambiguity? Here's an idea. If we have relative probabilities, then we can look at how sharp these probabilities are. Perhaps a better method is to simply look at the confusion matrix on the test set. The confusion matrix and the derived accuracy rates should correlate well with the level of class ambiguity. Plot these measures on a heat map.

**Comment by alasdairtran** [comment]
Another note on class ambiguity. Suppose that we have unbalanced classes in our training set. Then we can infer the proportion of the smaller classes.that need to be in the overlapping region, which we will feed into the classifier. This is where active learning might be of use, in addition to outlier detection.
